calc.py

```diff
@@ -24,10 +24,10 @@
 )
 
 # First year of tax year
-tax_year = 2019
-# Allowance is £12000 for 2019/20
+tax_year = 2020
+# Allowance for 2020/21
 # https://www.gov.uk/guidance/capital-gains-tax-rates-and-allowances#tax-free-allowances-for-capital-gains-tax
-capital_gain_allowance = 12000
+capital_gain_allowance = 12300
```

Comment:
Let's create a map for this with year as a key and include allowances for different years.

Reply:
Added suggestions